Formula/git-bug.rb

```diff
@@ -12,6 +12,17 @@ def install
   end
 
   test do
+    #Version
     assert_includes shell_output("#{bin}/git-bug --version"), "git-bug version"
+    #Version through git
+    assert_includes shell_output("git bug --version"), "git-bug version"
+    #List issues outside a repository
+    assert_includes shell_output("git bug ls 2>&1",1), "Error: git-bug must be run from within a git repo"
+    #Add issue outside a repository
+    assert_includes shell_output("git bug add 2>&1",1), "Error: git-bug must be run from within a git repo"
+    #Push issues to a remote from outside a repository
+    assert_includes shell_output("git bug push origin 2>&1",1), "Error: git-bug must be run from within a git repo"
+    #Pull issues from a remote from outside a repository
+    assert_includes shell_output("git bug pull origin 2>&1",1), "Error: git-bug must be run from within a git repo"
```

Comment:
I think we could even initialise a git repo for testing and exercise real result, like this:

https://github.com/Homebrew/homebrew-core/blob/39408092729cb66e30c8bbc2a6cf20e921ed46b2/Formula/hg-fast-export.rb#L49-L54

Reply:
Nice idea, done!

Reply:
(couldn't edit/close/etc those issues since they have an Id generated at runtime)